composer.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "e-moe/guzzle6-bundle",
+    "name": "tparcina/guzzle6-bundle",
```

Comment:
please, don't change package name

Reply:
Sorry. That was not in first commit. I'll change that. I though I head to do it in order to test my changes (to create a package in packagist), but at the end I have found out how test them without creating packagist package.